server/plugin.go

```diff
@@ -142,6 +149,11 @@ func (p *Plugin) generateDicePost(query, userID, channelID, rootID string) (*mod
 		text += fmt.Sprintf("\n- %s", strings.Join(formattedRollDetails, "\n- "))
 	}
 
+	// Display roll reason
+	if reason != "" {
+		text += fmt.Sprintf("\n```\n%s\n```", reason)
```

Comment:
I'm wondering, is it be better to show the roll reason before or after the roll? Eg. 
![image](https://github.com/user-attachments/assets/42f05377-4bc4-458b-b8be-a64834868c05)
 VS 
![image](https://github.com/user-attachments/assets/9ec2f562-33ad-49cd-8ff7-7b02a41544f4)

Reply:
I don't seem to care about this; it makes no difference to me. :)